components/k8s-model-server/http-proxy/app.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
What's the YAML for? Can we make it a ksonnet template? I think it would make sense to add a ksonnet template which has TF Serving and the proxy in a sideacar.

Reply:
i'll remove this and add following up ksonnet template